### PR TITLE
Makes Path `Home` Symbol Customizable

### PR DESCRIPTION
**Pull request by brucificus**
The makes the tilde (`~`) used in home-relative paths be replaceable by themes.

### CODE REVIEW

File: Helpers/Prompt.Tests.ps1 [hunk 1]
```diff
@@ -13,6 +13,7 @@ $global:ThemeSettings = New-Object -TypeName PSObject -Property @{
         SegmentSeparatorForwardSymbol  = '>'
         SegmentSeparatorBackwardSymbol = '<'
         PathSeparator                  = '\'
+        HomeSymbol                     = '*'
```
Comment (JanDeDobbeleer) [inline Helpers/Prompt.Tests.ps1:16]: Can you set this default value to `~`? That way it will stay consistent with current configs.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: Helpers/Prompt.Tests.ps1 (hunk 1 above)
Comment (FGS-BMarkham) [reply Helpers/Prompt.Tests.ps1]: My intent in deviating this value for the tests was to make sure the test verifies that the code under test respects the theme setting - and in turn does not, for example, have a regression wherein the implementation hard-codes the tilde.

What are your thoughts on such?

FWIW, my testing habits are very biased towards keeping tests from incidentally having or acquiring false positives. I might just be being overzealous with such.

Comment (brucificus) [reply Helpers/Prompt.Tests.ps1:16]: My intent in deviating this value for the tests was to make sure the test verifies that the code under test respects the theme setting - and in turn does not, for example, have a regression wherein the implementation hard-codes the tilde.

What are your thoughts on such?

FWIW, my testing habits are very biased towards keeping tests from incidentally having or acquiring false positives. I might just be being overzealous with such.

Comment (JanDeDobbeleer) [reply Helpers/Prompt.Tests.ps1:16]: Ah, no my bad I missed the fact this was added in the tests.

**Review by JanDeDobbeleer — APPROVED** [review]

**Comment by JanDeDobbeleer** [comment]
Can you rebase your branch on the latest master so the conflicts can be resolved?

**Comment by brucificus** [comment]
👍 Rebased.

**Pull request merged**